docs/source/Lines_functionality.ipynb

```diff
@@ -0,0 +1,1861 @@
+{
```

Comment:
<p>xarray -&gt; <code>xarray</code> or Xarray, for pandas you have it capitalized. You can decide on a way to represent the libraries and be consistent with it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>